docs/source/sci_api_tutorial/preparation_remove_hydrogens.rst

```diff
@@ -0,0 +1,149 @@
+==============================================
+ Preparation: Remove Hydrogens
+==============================================
+
+Briefs
+==============================================
+
+This science API, named ``enzy_htp.preparation.clean.remove_hydrogens``,
+removes hydrogen atoms from the supplied ``enzy_htp.structure.Structure`` class instance 
+(hereafter referred to as ``Structure`` instance).
+
+Input/Output
+==============================================
+
+.. panels::
+
+    :column: col-lg-12 col-md-12 col-sm-12 col-xs-12 p-2 text-left
+
+    .. image:: ../../figures/preparation_remove_hydrogens_dfd.svg
+        :width: 100%
+        :alt: preparation_remove_solvent
```

Comment:
Please do not have configs in this diagram. (so input data only) It will confuse users.

Reply:
Conducted minor modifications to diagrams.